dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/bnd.bnd

```diff
@@ -0,0 +1,29 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+
+fat.project: true
+
+tested.features:\
+	vmmapi-1.0
+
+-buildpath: \
+	commons-logging:commons-logging;version=1.1.3
```

Comment:
I would maybe remove all these except for the LDAP dependency so we can start fresh.

Reply:
Done